package.json

```diff
@@ -38,7 +40,9 @@
     "gulp-zip": "^3.0.2",
     "run-sequence": "^1.1.4",
     "semistandard": "^7.0.5",
-    "tape": "^4.4.0"
+    "tape": "^4.4.0",
+    "tape-expect": "^1.2.0",
```

Comment:
I don't know what best practice is, but I'd personally prefer having tests that validate values are equal rather than fancy expects. We should know exactly what the code is doing.